frontend/src/components/RecordsList.tsx

```diff
@@ -148,28 +154,38 @@ export default function RecordsList({
     }
   };
 
+  async function getExportLoader() {
+    if (selectedExportItem && selectedExportItem.customLoader) {
+      const data = await selectedExportItem.customLoader!();
+      return buildTsvString(
+        data,
+        selectedExportItem.columnDefs,
+        gridRef.current?.columnApi?.getAllGridColumns()
+      );
+    }
+    const { data } = await fetchMore({
+      variables: { offset: 0, limit: recordCount },
+    });
+    return buildTsvString(
+      data[recordsQueryName],
+      columnDefsForExport,
+      gridRef.current?.columnApi?.getAllGridColumns()
+    );
+  }
```

Comment:
If a custom export handler is present (e.g. run a custom query to download all anchor dates from Databricks), use that. Otherwise, use the existing exporting mechanism.